includes/shortcodes/class-wc-shortcode-checkout.php

```diff
@@ -229,6 +229,12 @@ private static function order_received( $order_id = 0 ) {
 		// Empty awaiting payment session.
 		unset( WC()->session->order_awaiting_payment );
 
+		// In case order is created from admin, but paid by the actual customer, store the ip address of the payer.
+		if ( false !== $order ) {
```

Comment:
Generally we just check for `$order` don't really need to enforce `false`, since `wc_get_order()` should already return false if the order does not exist.

Reply:
all righty, fixed.